firefox-ios/Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -547,6 +554,7 @@ class SearchViewController: SiteTableViewController,
 
     func loader(dataLoaded data: Cursor<Site>) {
         self.data = data
+        filteredBookmarksAndHistory = filterData(data: data)
```

Comment:
Do we need to keep the unfiltered bookmarks and history around for anything? If not, I wonder if it makes sense to just do:

```suggestion
        self.data = filterData(data: data)
```